app/main.py

```diff
@@ -16,4 +16,20 @@
 }
 collection_of_coins = {1, 2, 25}
 
-# write your code here
+
+def classify_variables(*args) -> dict:
```

Comment:
you don't need this function for this task, just write dict with keys and values already